edx_rest_framework_extensions/auth/jwt/cookies.py

```diff
@@ -30,4 +30,4 @@ def get_decoded_jwt(request):
 
     if not jwt_cookie:
         return None
-    return jwt_decode_handler(jwt_cookie)
+    return configured_jwt_decode_handler(jwt_cookie)
```

Comment:
**Reviewer Note:** This wasn't causing a known problem, but I updated it for consistency.  I did not add a special test for this.